Actions/RunPipeline/RunPipeline.ps1

```diff
@@ -140,7 +141,7 @@ try {
         }
         if (!$buildAll) {
             Write-Host "Get unmodified apps from baseline workflow run"
-            Get-UnmodifiedAppsFromBaselineWorkflowRun `
+            $downloadedAppsByType = Get-UnmodifiedAppsFromBaselineWorkflowRun `
```

Comment:
Can you also please leave a comment with your findings: that because the app files are downloaded in the build artifacts folders, the apps won't be built when calling `Run-AlPipeline`?